src/components/sessions/Sessions.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {SESSIONS_ITEMS} from "../../constatns";
+import Icon from "../Icons";
+import './_sessions.scss';
+
+function Sessions() {
+    return (
+        <section className="sessions section" id="sessions">
+            <div className="container">
+                <div className="section__heading">
+                <h2 className="section__title sessions__title">
+                    Sessions
+                    <span className="section__back-title">Sessions</span>
+                </h2>
+                </div>
+                <ul className="sessions__list list">
+                    {SESSIONS_ITEMS.map(item => (
+                        <li key={item.title} className="list__item">
+                            <div className="list__heading">
+                            <h4 className="list__title">{item.title} Session</h4>
+                            <div className="list__info">
+                                <div className="list__details">
+                                    <Icon className="list__icon" name="clock" />
+                                    <span>Duration: 6 hour</span>
```

Comment:
`Duration, Date, and Video (probably link) `has also come from the `SESSIONS_ITEMS`.